src/components/CreateMenu/BaseCreateMenu.js

```diff
@@ -0,0 +1,48 @@
+import React, {PureComponent} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import Popover from '../Popover';
+import styles from '../../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import MenuItem from '../MenuItem';
+import createMenuPropTypes from './CreateMenuPropTypes';
+
+const propTypes = {
+    // Callback fired when the menu is completely closed
+    onModalHide: PropTypes.func,
```

Comment:
I think this prop has a name that is incorrect according to the comment and the component name. I think `onHide` or `onMenuHide` would be more appropriate.

Reply:
I'll update it. It's not something that is exposed to the consumers `CreateMenu` though